### PR TITLE
Update Lokalise URL

### PR DESCRIPTION
**Pull request by wrt54g**
## Proposed change

I updated the Lokalise URL from lokalise.co to lokalise.com
## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [X] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [X] The code change is tested and works locally.
- [X] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: translations/README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
 # Translation Resources
-Translations are managed through [Lokalise](https://lokalise.co/). If you'd like to contribute, you can [join the project here](https://lokalise.co/signup/3420425759f6d6d241f598.13594006/all/). For more details, see our [translation guidelines documentation](https://developers.home-assistant.io/docs/en/internationalization_translation.html).
+Translations are managed through [Lokalise](https://lokalise.com/). If you'd like to contribute, you can [join the project here](https://lokalise.co/signup/3420425759f6d6d241f598.13594006/all/). For more details, see our [translation guidelines documentation](https://developers.home-assistant.io/docs/en/internationalization_translation.html).
```
Comment (frenck) [inline translations/README.md:2]: Lets just point to our developer documentation instead, as all links and details there are kept up to date.

Comment (wrt54g) [reply translations/README.md]: > Lets just point to our developer documentation instead, as all links and details there are kept up to date.

It's now updated to the developer docs

Comment (frenck) [reply translations/README.md]: the last URL needs an update as well

Comment (wrt54g) [reply translations/README.md]: > the last URL needs an update as well
The last URL is fixed too now.

**Review by frenck — APPROVED** [review]

**Pull request merged**